package.json

```diff
@@ -121,12 +122,17 @@
             },
             {
                 "command": "appSettings.Edit",
-                "title": "Edit Setting...",
+                "title": "Edit...",
```

Comment:
Minor: I'd suggest "Edit Value..."

Reply:
I believe in the context menu, with "Rename" next to it, the meaning of "Edit" should be clear enough. 🙂